index.js

```diff
@@ -27,6 +27,9 @@
 					result = new Date();
 					result.setTime(value.getTime());
 					return result;
+				} else if (value instanceof RegExp) {					
+					result = newRegExp(value);
+   					return result;
```

Comment:
Minor thing, can you fix up this indentation pls?

![](http://memecreator.org/static/images/memes/4099558.jpg)